packages/react-dom-bindings/src/server/fizz-instruction-set/ReactDOMFizzInstructionSetExternalRuntime.js

```diff
@@ -6,8 +6,11 @@ import {
   clientRenderBoundary,
   completeBoundary,
   completeSegment,
+  listenToFormSubmissionsForReplaying,
 } from './ReactDOMFizzInstructionSetShared';
 
+import {enableFormActions} from 'shared/ReactFeatureFlags';
```

Comment:
The test failures are because the hacky Fizz test runtime doesn't use the same bundling rules as we use when actually building this file so it doesn't know how to resolve the feature flags even though the prod builds do.

Reply:
For now, you could gate the tests to only run during build, like this:

```js
// @gate !source
```

Reply:
I have an alternative strategy here https://github.com/facebook/react/pull/26717 that I think is probably the better solution anyway.